projects/plugins/jetpack/modules/subscriptions/subscribe-modal/subscribe-modal.js

```diff
@@ -2,20 +2,22 @@ const { domReady } = wp;
 
 domReady( function () {
 	const modal = document.getElementsByClassName( 'jetpack-subscribe-modal' )[ 0 ];
-	const close = document.getElementsByClassName( 'jetpack-subscribe-modal__close' )[ 0 ];
 
 	if ( ! modal ) {
 		return;
 	}
 
+	const close = document.getElementsByClassName( 'jetpack-subscribe-modal__close' )[ 0 ];
+	const modalDismissedCookie = 'jetpack_subscribe_modal_dismissed';
+	const hasModalDismissedCookie = document.cookie.indexOf( modalDismissedCookie ) > -1;
```

Comment:
I think `document.cookie` always exists, but I wanted to mention in case we need some protection, e.g. `document.cookie?.indexOf()`

Reply:
I've added a check for this. For some reason, adding the optional chaining operator was giving an eslint error.